07/src/academy/pocu/comp2500samples/w07/graphics/Circle.java

```diff
@@ -0,0 +1,29 @@
+package academy.pocu.comp2500samples.w07.graphics;
+
+public class Circle extends Graphic {
+    private Point center;
+    private int radius;
+
+    public Circle(

```

Comment:
is thi sreally normal to put first param in the new line? i always thought u would show first parameter on the same line at least